internal/gensupport/resumable.go

```diff
@@ -36,6 +37,11 @@ const (
 	// should be retried.
 	// https://cloud.google.com/storage/docs/json_api/v1/status-codes#standardcodes
 	statusTooManyRequests = 429
+
+	// statusRequestTimeout is returned by the storage API if the request times
+	// out on the server side. The request should be retried.
+	// https://cloud.google.com/storage/docs/json_api/v1/status-codes#standardcodes
```

Comment:
For storage this is documented safe to retry, but I do wonder if it is in generate. I am thinking of a case where your request gets to the backend, is fully processed, but the response is never received back on the client side. If an API is not idempotent the second request may not yield the expected result. Thoughts?

Reply:
Good point-- in any case this error has not be a major focus of complaints so far (for writes in the go libraries anyway). I could drop this one for now if you think it's better.

Reply:
I think we would want to dive into this more before adding holistically. We should probably only generically retry things we have documented in one of our aips. This is the only one I found with a quick search that mentions retry codes: https://google.aip.dev/194

Reply:
Makes sense-- I removed the retry on 408.